icefall/utils.py

```
@@ -670,14 +673,14 @@ def store_translations(
             hyp_list.append(hyp)
             ref_list.append(ref_tgt)
 
-    with open(bleu_file, 'w') as b:
+    with open(bleu_file, "w") as b:
         print(str(bleu.corpus_score(hyp_list, [ref_list])), file=b)
         print(f"BLEU signiture: {str(bleu.get_signature())}", file=b)
```

Ruff (0.15.18)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)